index.html

```diff
@@ -13,25 +13,31 @@
 <body>
     <header class="header-bar d-flex">
         <a class="header-title" href="#">Syed Aman</a>
-        <a class="menu-button" href="#" onclick="toggle()">
+        <nav class="menu-button"  onclick="toggle()">
             <img id="hamburger-btn" src="assets/ham.png" alt="hamburger">
-        </a>
+        </nav>
     </header>
 
     <section class="padding-section d-flex dark-section hero-section">
         <h1 class="section-title">Hey There.<br>I'm Aman</h1>
-        <p class="section-subtitle">I'm a software developer</p>
+        <h2 class="section-subtitle">I'm a software developer</h2>
         <p class="section-text hero-section-text">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
         <ul class="d-flex social-media hero-social-media">
             <li>
-                <a rel="noopener" href="https://github.com/SyedAman10" target="_blank"><img class="icon-link" src="assets/github.png" alt="Link to GitHub profile"></a>
+                <a rel="noopener" href="https://github.com/SyedAman10" target="_blank"><i class="fa fa-github " aria-hidden="true"></i></a>
             </li>
             <li>
-                <a rel="noopener" href="https://www.linkedin.com/in/syed-aman-ullah-238500196/" target="_blank"><img class="icon-link" src="assets/link.png" alt="Link to Linkedin profile"></a>
+                <a rel="noopener" href="https://www.linkedin.com/in/syed-aman-ullah-238500196/" target="_blank"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
             </li>
             <li>
-                <a rel="noopener" href="https://twitter.com/SyedAman9248" target="_blank"><img class="icon-link" src="assets/twitter.png" alt="Link to Twitter profile"></a>
+                <a href="#" class="icon3"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+            </li>
+            <li>
+                <a rel="noopener" href="https://twitter.com/SyedAman9248" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li>
+                <a href="#" class="icon5"><i class="fa fa-medium" aria-hidden="true"></i></a>
             </li>
         </ul>
```

Comment:
- [ ] One of the major requirements is that you make your design look exactly like that of the Figma's Template that you chose. Currently, it doesn't look like it and it would be nice if you implemented what the previous code reviewer suggested as regards the social media icon by aligning them more to the left than the center as you have it.

## Your Design 

![image](https://user-images.githubusercontent.com/77106962/144461959-8b1c25d8-aa63-4fb6-98d1-0b88d14cddb4.png)

## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144462052-56274968-77a9-4d71-af83-6879d7f7966e.png)

#### I will like to recommend that you kindly use the margins used for your template. 🙏